requirements/doc-requirements.txt

```diff
@@ -5,3 +5,6 @@ sphinx==3.5.4
 jinja2==3.0.3
 sphinx-autobuild
 sphinx-click
+tensorflow
+pyspark
+datasets
```

Comment:
Do we need these for building docs?

Reply:
Yeah, we need these for the docs build because, otherwise, the `mlflow.data.from_tensorflow`, etc. methods are not defined